src/page components/CurrenciesListPage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
please name the folder `views` or `pages` and do not use empty space ` ` in `file` or `directory` names, it's an anty-pattern  😄